include/verilated_funcs.h

```diff
@@ -1526,6 +1526,16 @@ static inline WDataOutP VL_STREAML_WWI(int lbits, WDataOutP owp, WDataInP const
     return owp;
 }
 
+template <typename T>
+static inline void VL_ASSIGN_DQ(VlQueue<T>& q, int elem_size, int lbits, QData fromp) {
```

Comment:
"D" implies a double, so think we need a more verbose name.

```suggestion
static inline void VL_ASSIGN_DYN_Q(VlQueue<T>& q, int elem_size, int lbits, QData fromp) {
```